autogpt_platform/frontend/src/app/(platform)/auth/auth-code-error/page.tsx

```diff
@@ -23,15 +28,105 @@ export default function AuthErrorPage() {
   }, []);
 
   if (!errorType && !errorCode && !errorDescription) {
-    return <div>Loading...</div>;
+    return (
+      <div className="flex h-screen items-center justify-center">
+        <Text variant="body">Loading...</Text>
```

Comment:
we have loading spinner component

Reply:
its legacy so we shouldn't include it in new code